src/Codec/Xlsx/Types.hs

```diff
@@ -48,6 +50,10 @@ module Codec.Xlsx.Types (
     , Cell.cellStyle
     , Cell.cellComment
     , Cell.cellFormula
+    -- ** Row properties
+    , rowHeightLens
+    , _CustomHeight
+    , _AutomaticHeight
```

Comment:
Should I move these to a seperate PR instead?

Reply:
I don't think it's required when it's already here. Rather a note that it would be better to have that as a separate PR but I'm OK with merging it in this one as well.